tools/deb/postinst.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Copyright 2017 Istio Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+set -e
+
+action="$1"
+oldversion="$2"
+
+umask 022
+
+if ! getent passwd istio-proxy >/dev/null; then
+    addgroup --system istio-proxy
```

Comment:
call it istio please. (like Docker, which just has a docker user.. no docker-daemon/docker-x.)

Reply:
that quite different. we may run proxy and mixer on same machine as different users